doc/source/reference/maskedarray.generic.rst

```diff
@@ -379,8 +379,8 @@ is masked.
 When accessing a slice, the output is a masked array whose
 :attr:`~MaskedArray.data` attribute is a view of the original data, and whose
 mask is either :attr:`nomask` (if there was no invalid entries in the original
-array) or a copy of the corresponding slice of the original mask. The copy is
-required to avoid propagation of any modification of the mask to the original.
```

Comment:
Should note the previous behavior, the new behavior, and the release in which things changed.